modules/transport.js

```diff
@@ -522,6 +522,25 @@ Transport.prototype.onMessage = function (msg, broadcast) {
 	}
 };
 
+/**
+ * Broadcasts the passed block using __private.broadcaster.broadcast
+ * @implements {Broadcaster.broadcast}
+ * @param {Object} params
+ * @param {Object} block
+ */
+Transport.prototype.broadcastBlock = function (params, block) {
+	if (!__private.broadcaster.maxRelays(block)) {
+		library.logger.info('Broadcasting the block of id: ' + block.id);
+		library.logger.trace('Broadcasting the block parameters', + params);
+		__private.broadcaster.broadcast(params, {
+			api: '/blocks',
+			data: {block: block},
+			method: 'POST',
```

Comment:
We need to unify the call, broadcast is been called in a single line in other parts of the code.

Reply:
Ok, will do that.